shell/edit/__tests__/management.cattle.io.clusterroletemplatebinding.test.ts

```diff
@@ -25,7 +25,7 @@ describe('view: management.cattle.io.clusterroletemplatebinding should', () => {
           'i18n/t':                  (val) => val,
           'i18n/exists':             jest.fn(),
         },
-        dispatch: { 'management/findAll': () => ([]) }
+        dispatch: jest.fn(),
```

Comment:
I would rather mock single keys, because it helps to understand what are we doing and because it's easier to edit afterward.
I completely missed that we did also in the other test. This is quite crucial to make it easier for other components, like in this case. E.g. think about how much time you had to spent to figure it out 😬 

Replacement should be something like:

```ts
dispatch: {
  'management/findAll': () => ([]),
  'cru-resource/setCreateNamespace': jest.fn()
}
```

Reply:
Hmm, it seems that the suggested approach still fails in the same manner:

```
  ● view: management.cattle.io.clusterroletemplatebinding should › should only show one error banner

    TypeError: this.$store.dispatch is not a function
```

I'm unsure what the difference is at this point that's causing the error, but it seemed pretty clear to me that `this.$store.dispatch` is lacking a proper mock. I tried updating this so that we could retain individual keys without failing the test, but I'm still not too satisfied with it. 

I tried looking to the vue test utils docs for guidance[^1], but we utilize a different pattern for working with Vuex than what they recommend. 

[^1]: https://v1.test-utils.vuejs.org/guides/using-with-vuex.html#mocking-actions

Reply:
We have already a case for the `dispatch()`, let me check it out.

Reply:
Ehm, sorry I got confused with the `getter()` 🤦